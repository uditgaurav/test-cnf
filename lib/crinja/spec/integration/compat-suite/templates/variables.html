<html>
  <head>
    <title>{{ product.name }}</title>
  </head>
  <body>
    <h1>{{ product.name }} - {{ product.manufacturer }}</h1>
    <p>{{ product.summary }}</p>
    <p>Â£{{ (product.price * 1.20) | int }} (VAT inc.)</p>
    <p>Look at reviews from your friends {{ username }}</p>
    <button>Buy!</button>
  </body>
</html>
