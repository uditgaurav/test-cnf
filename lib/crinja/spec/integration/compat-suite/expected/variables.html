<html>
  <head>
    <title>Moto G</title>
  </head>
  <body>
    <h1>Moto G - Motorala</h1>
    <p>A phone</p>
    <p>Â£120 (VAT inc.)</p>
    <p>Look at reviews from your friends bob</p>
    <button>Buy!</button>
  </body>
</html>